### PR TITLE
[Refactor] Refactor config name for 1.x.

### PR DESCRIPTION
**Pull request by pixeli99**

### CODE REVIEW

File: configs/mot/bytetrack/README.md [hunk 1]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+|  Method   | Detector |           Train Set           |    Test Set    | Public | Inf time (fps) | HOTA | MOTA | IDF1 |  FP   |  FN   | IDSw. |                                        Config                                        |                                                                                                                                                           Download                                                                                                                                                           |
+| :-------: | :------: | :---------------------------: | :------------: | :----: | :------------: | :--: | :--: | :--: | :---: | :---: | :---: | :----------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:35]: test review

File: configs/mot/bytetrack/README.md [hunk 2]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
-| ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT20-test  |   N    |       -        | 61.5 | 77.0 | 75.4 | 33,083 | 84,433 | 1,345 | [config](bytetrack_yolox_x_crowdhuman_mot20-private.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
+|  Method   | Detector |        Train Set         |  Test Set   | Public | Inf time (fps) | HOTA | MOTA | IDF1 |   FP   |   FN   | IDSw. |                                    Config                                     |                                                                                                                                                      Download                                                                                                                                                      |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:45]: test

**Review by JingweiZhang12 — COMMENTED** [review]
test

File: configs/mot/bytetrack/README.md [hunk 3]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| :-------: | :------: | :---------------------------: | :------------: | :----: | :------------: | :--: | :--: | :--: | :---: | :---: | :---: | :----------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT17-half-train | MOT17-half-val |   N    |       -        | 67.7 | 78.6 | 79.2 | 12909 | 21024 |  666  | [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot17halftrain_test-mot17halfval.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:36]: test

Comment (JingweiZhang12) [reply configs/mot/bytetrack/README.md:36]: reply

**Review by JingweiZhang12 — COMMENTED** [review]
test

File: configs/mot/bytetrack/README.md [hunk 4]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT17-half-train |   MOT17-test   |   N    |       -        | 61.7 | 78.1 | 74.8 | 36705 | 85032 | 2049  |  [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot17halftrain_test-mot17test.py)   | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:37]: edit action test

Comment (JingweiZhang12) [reply configs/mot/bytetrack/README.md:37]: test reply

File: configs/mot/bytetrack/README.md [hunk 5]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| :-------: | :------: | :----------------------: | :---------: | :----: | :------------: | :--: | :--: | :--: | :----: | :----: | :---: | :---------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT17-train |   N    |       -        | 57.3 | 64.9 | 71.8 | 33,747 | 83,385 | 1,263 | [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot20train_test-mot20test.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:47]: reply

**Comment by JingweiZhang12** [comment]
test submitted

**Comment by JingweiZhang12** [comment]
test

**Review by JingweiZhang12 — COMMENTED** [review]
test

File: configs/mot/bytetrack/README.md [hunk 6]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT17-half-train | MOT17-half-val |   N    |       -        | 67.7 | 78.6 | 79.2 | 12909 | 21024 |  666  | [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot17halftrain_test-mot17halfval.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT17-half-train |   MOT17-test   |   N    |       -        | 61.7 | 78.1 | 74.8 | 36705 | 85032 | 2049  |  [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot17halftrain_test-mot17test.py)   | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500.log.json) |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:38]: test

File: configs/mot/bytetrack/bytetrack_yolox-x_8x4bs-amp-80e_crowdhuman-mot17halftrain_test-mot17halfval.py [hunk 7]
```diff
@@ -1,4 +1,7 @@
-_base_ = ['./bytetrack_yolox_x_crowdhuman_mot20-private.py']
+_base_ = [
+    './bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot17halftrain_'
+    'test-mot17halfval.py'
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/bytetrack_yolox-x_8x4bs-amp-80e_crowdhuman-mot17halftrain_test-mot17halfval.py:3]: test

File: configs/mot/bytetrack/README.md [hunk 8]
```diff
@@ -31,18 +31,18 @@ Please note that the performance on `MOT17-half-val` is comparable with the perf
...
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT17-train |   N    |       -        | 57.3 | 64.9 | 71.8 | 33,747 | 83,385 | 1,263 | [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot20train_test-mot20test.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
+| ByteTrack | YOLOX-X  | CrowdHuman + MOT20-train | MOT20-test  |   N    |       -        | 61.5 | 77.0 | 75.4 | 33,083 | 84,433 | 1,345 | [config](bytetrack_yolox-x_8x4bs-80e_crowdhuman-mot20train_test-mot20test.py) | [model](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040-9ce38a60.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot20-private_20220506_101040.log.json) |
```
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:48]: fa

**Review by JingweiZhang12 — COMMENTED** [review]
test

File: configs/mot/bytetrack/README.md (hunk 1 above)
Comment (JingweiZhang12) [inline configs/mot/bytetrack/README.md:35]: test

File: configs/vid/fgfa/fgfa_faster_rcnn-resnet101-dc5_8x1bs-7e_imagenetvid.py [hunk 9]
```diff
@@ -1,4 +1,4 @@
-_base_ = ['./selsa_troialign_faster_rcnn_r50_dc5_7e_imagenetvid.py']
+_base_ = ['./fgfa_faster_rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py']
```
Comment (GT9505) [inline configs/vid/fgfa/fgfa_faster_rcnn-resnet101-dc5_8x1bs-7e_imagenetvid.py:1]:
```suggestion
_base_ = ['./fgfa_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py']
```

modify it for all config & readme.md & metafile.yml of FGFA and SELSA

File: configs/vid/temporal_roi_align/metafile.yml [hunk 10]
```diff
@@ -13,9 +13,9 @@ Collections:
     README: configs/vid/temporal_roi_align/README.md
 
 Models:
-  - Name: selsa_troialign_faster_rcnn_r50_dc5_7e_imagenetvid
+  - Name: selsa_troialign_faster_rcnn-resnet50-dc5_8x1bs-7e_imagenetvid
```
Comment (GT9505) [inline configs/vid/temporal_roi_align/metafile.yml:16]:
```suggestion
  - Name: selsa-troialign_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid
```

File: tests/test_models/test_vid/test_fgfa.py [hunk 11]
```diff
@@ -18,7 +18,7 @@ def setUpClass(cls):
         register_all_modules()
 
     @parameterized.expand([
-        'vid/fgfa/fgfa_faster_rcnn_r50_dc5_7e_imagenetvid.py',
+        'vid/fgfa/fgfa_faster_rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py',
```
Comment (GT9505) [inline tests/test_models/test_vid/test_fgfa.py:21]:
```suggestion
        'vid/fgfa/fgfa_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py',
```

File: tests/test_models/test_vid/test_selsa.py [hunk 12]
```diff
@@ -18,21 +18,21 @@ def setUpClass(cls):
         register_all_modules()
 
     @parameterized.expand([
-        'vid/selsa/selsa_faster_rcnn_r50_dc5_7e_imagenetvid.py',
+        'vid/selsa/selsa_faster_rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py',
```
Comment (GT9505) [inline tests/test_models/test_vid/test_selsa.py:21]:
```suggestion
        'vid/selsa/selsa_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py',
```

File: tests/test_models/test_vid/test_selsa.py [hunk 13]
```diff
@@ -18,21 +18,21 @@ def setUpClass(cls):
...
         'vid/temporal_roi_align/'
-        'selsa_troialign_faster_rcnn_r50_dc5_7e_imagenetvid.py'
+        'selsa_troialign_faster_rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py'
```
Comment (GT9505) [inline tests/test_models/test_vid/test_selsa.py:23]:
```suggestion
        'selsa-troialign_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py'
```

File: configs/mot/qdtrack/metafile.yml [hunk 14]
```diff
@@ -13,9 +13,9 @@ Collections:
     README: configs/mot/qdtrack/README.md
 
 Models:
-  - Name: qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half
+  - Name: qdtrack_faster-rcnn_resnet50-fpn_8x2bs-4e_mot17halftrain_test-mot17halfval
```
Comment (GT9505) [inline configs/mot/qdtrack/metafile.yml:16]:
```suggestion
  - Name: qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain_test-mot17halfval
```

**Comment by GT9505** [comment]
Overrall, the config of FGFA, SELSA, SELSA+TROI, QDtrack need to be renamed

File: configs/mot/bytetrack/README.md (hunk 8 above)
Comment (GT9505) [inline configs/mot/bytetrack/README.md:48]: all configs in the readme.md should contain `amp`, since they are trained using fp16

Comment (pixeli99) [reply configs/mot/bytetrack/README.md:48]: I also modified the file name in metafile. Is it necessary to change it？

Comment (pixeli99) [reply configs/mot/bytetrack/README.md:48]: For other parts, I just made some changes about `-`.

File: configs/mot/qdtrack/metafile.yml [hunk 15]
```diff
@@ -13,9 +13,9 @@ Collections:
     README: configs/mot/qdtrack/README.md
 
 Models:
-  - Name: qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half
+  - Name: qdtrack/qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain_test-mot17halfval
```
Comment (GT9505) [inline configs/mot/qdtrack/metafile.yml:16]:
```suggestion
  - Name: qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain_test-mot17halfval
```

File: configs/mot/qdtrack/qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_crowdhuman-mot17halftrain_test-mot17halfval.py [hunk 16]
```diff
@@ -1,4 +1,7 @@
-_base_ = ['./qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py']
+_base_ = [
+    './qdtrack_faster-rcnn_resnet50-fpn_8x2bs-4e_mot17halftrain_'
```
Comment (GT9505) [inline configs/mot/qdtrack/qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_crowdhuman-mot17halftrain_test-mot17halfval.py:2]:
```suggestion
    './qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain_'
```

File: tests/test_models/test_mot/test_qdtrack.py [hunk 17]
```diff
@@ -20,7 +20,8 @@ def setUpClass(cls):
         register_all_modules(init_default_scope=True)
 
     @parameterized.expand([
-        'mot/qdtrack/qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py',
+        'mot/qdtrack/qdtrack_faster-rcnn_resnet50-fpn_8x2bs-4e_mot17halftrain_'
```
Comment (GT9505) [inline tests/test_models/test_mot/test_qdtrack.py:23]:
```suggestion
        'mot/qdtrack/qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain_'
```

File: tests/test_models/test_mot/test_qdtrack.py [hunk 18]
```diff
@@ -30,8 +31,8 @@ def test_qdtrack_init(self, cfg_file):
         assert model.track_head
 
     @parameterized.expand([
-        ('mot/qdtrack/qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py',
-         ('cpu', 'cuda')),
+        ('mot/qdtrack/qdtrack_faster-rcnn_resnet50-fpn_8x2bs-4e_mot17halftrain'
```
Comment (GT9505) [inline tests/test_models/test_mot/test_qdtrack.py:34]:
```suggestion
        ('mot/qdtrack/qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain'
```

File: tests/test_models/test_mot/test_qdtrack.py [hunk 19]
```diff
@@ -66,8 +67,8 @@ def test_qdtrack_forward_loss_mode(self, cfg_file, devices):
             assert isinstance(losses, dict)
 
     @parameterized.expand([
-        ('mot/qdtrack/qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py',
-         ('cpu', 'cuda')),
+        ('mot/qdtrack/qdtrack_faster-rcnn_resnet50-fpn_8x2bs-4e_mot17halftrain'
```
Comment (GT9505) [inline tests/test_models/test_mot/test_qdtrack.py:70]:
```suggestion
        ('mot/qdtrack/qdtrack_faster-rcnn-resnet50-fpn_8x2bs-4e_mot17halftrain'
```

**Review by GT9505 — APPROVED** [review]

**Pull request merged**